### ISSUE TITLE
More details about NuScenes dataset

### ISSUE THREAD

**Issue by 3bobo** [Ghostish/BAT#1]
Thanks for your work! I want to ask for some information about NuScenes dataset.
1. you said you use the training set of NuScenes for training and test on its validation set. Have you used the two splits all? As I know, there are some tracking sequences whose first frame has no points, do you test them too?
2. I see you test Car, Truck, Trailer, Bus in NuScenes, how about the other categories? such as "pedestrian", "motorcycle"?
Thanks again for your work!

**Comment by Ghostish** [comment]
Thanks for your interest. NuScenes is a more challenging dataset and there is still a lot for us to work on.

1. For the NuScenes dataset, there are lots of sequences where the first object BB has no points. We do not consider such sequences because an empty template does not give any useful information to the tracker. Tracking on such sequences makes no sense.
2. For the other classes, the number of sequences that meet the above condition is too small to achieve effective training.  We didn't include those results in our paper due to severe underfitting.

**Comment by 3bobo** [comment]
Thanks for your reply! It is indeed not feasible to track the sequence with empty points in the first frame. Furthermore, could you give more detailed conditions to choose the test sequence? Such as tracking length or others

**Comment by Ghostish** [comment]
Hi there.   More details about NuScenes can be checked from the codes.  We are refactoring our codes and will release them as soon as possible.

**Comment by 3bobo** [comment]
Ok, waiting for your code. Thanks!
